solace-message-router/nested/workspace-disabled-shared-resources.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
can we just remove this pattern and use the condition property?

Reply:
I could not figure out how to populate the workspaceId and workspaceKey settings in the OmsAgentForLinux resource using conditions. I was unable to get conditions to work with the following reference functions:

        "settings": {
          "workspaceId": "[reference(variables('solaceWorkspaceName')).outputs.workspaceId.value]"
        },
        "protectedSettings": {
          "workspaceKey": "[reference(variables('solaceWorkspaceName')).outputs.workspaceKey.value]"
        }

Even if the deployment of the OmsAgentForLinux resource is wrapped in a condition, if the workspaceId and workspaceKey output values are not set because the reference to the shared template does not exist when the template is initially launched, template validation throws an exception.

Forum posts on the topic seem to suggest this is not possible. Would you have some hints to work around this issue?